### ISSUE TITLE
[kickasstorrent] Exception (kickasstorrent): Request headers must contain only ASCII characters.: Request headers must contain only ASCII characters. (Test)

### ISSUE THREAD

**Issue by matthieuneier** [Jackett/Jackett#5594]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.

If you are experiencing an issue with a tracker, then:
1. Use your browser to check you can access the site directly, and if a login is required, check you can login and that you do not have any outstanding account issues.
2. If you haven't already, try upgrading to the latest version of Jackett, your issue may have already been resolved.
3. If it is still not working for you, then a **full enhanced log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

(At the bottom of your Jackett Dashboard page) **Jackett version**:  
(if you installed using Jackett.Binaries.Mono.tar.gz) **Mono version**:

-------------------------------

**Comment by matthieuneier** [comment]
Jackett Version 0.11.463.0

ett indexer in Sonarr or Radarr
Go to Settings > Indexers > Add > Torznab > Custom.
Click on the indexers corresponding  button and paste it into the Sonarr/Radarr URL field.
For the API key use djk6wuehsbmu8hfpvbyj5b1m1ga8m8rd.
Configure the correct category IDs via the (Anime) Categories options. See the Jackett indexer configuration for a list of supported categories.
Adding a Jackett indexer in CouchPotato
Go to Settings > Searchers.
Enable TorrentPotato.
Click on the indexers corresponding button and paste it into the CouchPotato host field.
For the Passkey use djk6wuehsbmu8hfpvbyj5b1m1ga8m8rd. Leave the username field blank.
Adding a Jackett indexer to RSS clients (RSS feed)
Click on the indexers corresponding button and paste it into the URL field of the RSS client.
You can adjust the q (search string) and cat (categories) arguments accordingly. E.g. ...&cat=2030,2040&q=big+buck+bunny will search for "big buck bunny" in the Movies/SD (2030) and Movies/HD (2040) categories (See the indexer configuration for available categories).
Jackett Configuration
 

Admin password: 
••••••••••
 
Base Path Override: 
/jackett
Server port: 
9117
Manual download blackhole directory: 
c:\torrents\
Proxy type: 
Proxy url: 
Blank to disable
Proxy port: 
Proxy username: 
admin
Proxy password: 
External access:   
Disable auto update:   
Update to pre-release:   
Enhanced logging:   
OMDB API key: 
OMDB API Url: 
Blank for default
Jackett Version 0.11.463.0
×
Server Logs
When	Level	Message
Jul 05, 2019 07:57:30 AM	Error	
Jackett.Common.IndexerException: Exception (kickasstorrent): Request headers must contain only ASCII characters. ---> System.Net.Http.HttpRequestException: Request headers must contain only ASCII characters.
   at System.Net.Http.HttpConnection.WriteStringAsync(String s)
   at System.Net.Http.HttpConnection.WriteHeadersAsync(HttpHeaders headers, String cookiesFromContainer)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at CloudFlareUtilities.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241
   at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 114
   at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 445
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1230
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 331
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 351
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 815
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/appveyor/projects/jackett/src/Jackett.Common/Services/IndexerManagerService.cs:line 222
   at Jackett.Server.Controllers.IndexerApiController.Test() in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 128
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 29

**Comment by sn33ky** [comment]
I have the same problem.

I filled in the cookie from kickasstorrents: 
**Cookie:** `katcr_ses_=a%3A4%3A%7Bi%3A0%3Bi%3A245659%3Bi%3A1%3Bs%3A40%3A%227104dee387f21fba883b7d773804c8856220185d%22%3Bi%3A2%3Bi%3A1562310935%3Bi%3A3%3Bi%3A0%3B%7D; KATSSESS_ID70AT8=iiqu4334at22i6nitjokul3apvchgsv1`

**Only the part:** `=a%3A4%3A%7Bi%3A0%3Bi%3A245659%3Bi%3A1%3Bs%3A40%3A%227104dee387f21fba883b7d773804c8856220185d%22%3Bi%3A2%3Bi%3A1562310935%3Bi%3A3%3Bi%3A0%3B%7D`

Is that OK?

_Jackett Version 0.11.420.0_

**Error:**
Jackett.Common.IndexerException: Exception (kickasstorrent): Request headers must contain only ASCII characters. ---> System.Net.Http.HttpRequestException: Request headers must contain only ASCII characters.
   at System.Net.Http.HttpConnection.WriteStringAsync(String s)
   at System.Net.Http.HttpConnection.WriteHeadersAsync(HttpHeaders headers, String cookiesFromContainer)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at CloudFlareUtilities.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241
   at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 114
   at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 445
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1230
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 331
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 351
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 815
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/appveyor/projects/jackett/src/Jackett.Common/Services/IndexerManagerService.cs:line 222
   at Jackett.Server.Controllers.IndexerApiController.Test() in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 128
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 29

**Comment by garfield69** [comment]
no. make sure you are getting the cookie from the _request headers_ section of the _Headers_ tab, and not from the _Cookies_ tab.
then copy everything from the right of the **Cookies:** string and paste it into the box in the config.

**Comment by sn33ky** [comment]
i follow the guide and get this result: 
![image](https://user-images.githubusercontent.com/78500/60704850-e492f580-9f05-11e9-8603-21b6ee62934d.png)
Or am i missing something? 

Now i pasted the string: 
`katcr_ses_=a%3A4%3A%7Bi%3A0%3Bi%3A245659%3Bi%3A1%3Bs%3A40%3A%227104dee387f21fba883b7d773804c8856220185d%22%3Bi%3A2%3Bi%3A1562310935%3Bi%3A3%3Bi%3A0%3B%7D; KATSSESS_ID70AT8=iiqu4334at22i6nitjokul3apvchgsv1`

But still the same error: 
`Jackett.Common.IndexerException: Exception (kickasstorrent): Request headers must contain only ASCII characters. ---> System.Net.Http.HttpRequestException: Request headers must contain only ASCII characters.
   at System.Net.Http.HttpConnection.WriteStringAsync(String s)
   at System.Net.Http.HttpConnection.WriteHeadersAsync(HttpHeaders headers, String cookiesFromContainer)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at CloudFlareUtilities.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241
   at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 114
   at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 445
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1230
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 331
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 351
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 815
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/appveyor/projects/jackett/src/Jackett.Common/Services/IndexerManagerService.cs:line 222
   at Jackett.Server.Controllers.IndexerApiController.Test() in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 128
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 29`

**Comment by garfield69** [comment]
works for me, what can I say.
you appear to have the correct info. 
are you sure that the box is empty of any previous content before you paste the new cookie in?
i'm using windows 10 chrome browser, what are you using?

**Comment by garfield69** [comment]
I even pasted your cookie into my config and was able to use the session!

**Comment by sn33ky** [comment]
I'm using google chrome Versie 75.0.3770.100 (Officiële build) (64-bits) from a Windows 10 pro machine.

I tried emtying the box, and afterwards put the string in after first copying it to a notepad.
![image](https://user-images.githubusercontent.com/78500/60706392-a5ff3a00-9f09-11e9-9514-2b6a42abab5f.png)

But still the same error. If you need more info to troubleshoot, or if i can help. let me know.

**Comment by garfield69** [comment]
have you copied the string straight from the select/copy to the paste without the notepad in between?

what about deleting the kat config from the dashboard, shutting down the jackett service, restarting and adding the kat config afresh?

if this does not work there  is nothing I can do for you, its beyond my ken as to why it does not work for you. as I said in an earlier post, I can even make your cookie work for me.

If you still cannot got it to work let me know and I will assign it to the c# devs, although they do not have any free time at the moment so don't expect a prompt response :-(

**Comment by sn33ky** [comment]
> have you copied the string straight from the select/copy to the paste without the notepad in between?
Yup. but i was hoping the notepad do the trick because of an parsing error or something about the ASCII error
> what about deleting the kat config from the dashboard, shutting down the jackett service, restarting and adding the kat config afresh?
I tried that but stil lthe same error.

No problem i will delete it and wait till some1 can look @ it. Luckely there are a bunch of good torrent providers :)

**Comment by ToXinE** [comment]
I think there is a problem in the cookie : as the message state, it's not in a correct ascii format as you can see : 
![image](https://user-images.githubusercontent.com/6205053/60723665-892c2c00-9f34-11e9-8bf5-3a6c1bfb7df2.png)

**Comment by Arzte** [comment]
I have the same issue, and when first entering the cookie (whether it's mine or the one @sn33ky provided for troubleshooting, it "successfully configures" it, but when you got to test it, then you get the same error listed in the above comments. 

@ToXinE I'm not so sure, when you add that random garbage in the cookie string, it doesn't error out the same, but without that garbage, you can "save" the cookie just fine, it's just when you go to test it, the same error happens regardless.

**Comment by garfield69** [comment]
all of you with this problem, please post the platform you are running jackett on (linux, ubunto, win, pi, mojave, etc) and the package you installed (ARMv7, AMD64, MacOS, docker, etc)
Also specify if you are using the jackett default settings, or if you have enabled options (sockets, proxy, different http client with the -c or --useClient CLI options, something else?).
lets see if there is trend.

**Comment by joridiculous** [comment]
this is ridiculous. Getting same error, cookie or no cookie.
(with a cookie)  btw: Since When was this cookie a thing?
https://imgur.com/s3YuKnt 
testing gives this
https://imgur.com/FbM4fzP

**Comment by garfield69** [comment]
@joridiculous  the cookie login is a method used to bypass regular logins that have a captcha to prove-you-are-human which jackett may not have a solution for.

mind you, isn't KickAssTorrent (katcr.co) a public site? what are we login in for?

Seems I can search and download magnets from the site without login.
kaso17 set a cookie login for the indexer back in Jan 2018, I've no idea why, possibly back then there was some anti-DDoS checking os something similar. Does not appear to be required at the moment.

login dropped beginning with jackert pre-release 0.11.472, let see how it goes.

**Comment by garfield69** [comment]
Oh, and before you report that its not working, be aware that the site is undergoing maintenance at the moment, no ETA for completion.
![image](https://user-images.githubusercontent.com/6933240/60762321-29a75b00-a0b1-11e9-9f87-3835fab9e15a.png)

**Comment by ToXinE** [comment]
the site wasn't in maintenance at the time of the issue

**Comment by garfield69** [comment]
the site is back up and running now.

**Comment by sn33ky** [comment]
Does this mean the problem is also solved. As the status from this bug report is "solved"?

**Comment by garfield69** [comment]
yes it should be.
if you are on the latest jackett v0.11.475, the kickasstorrent indexer no longer attempts to login to the site. 
as long as https://katcr.co/ is accessible and the site is operational you should be good to go.

**Comment by tbotcotw** [comment]
This is still an issue. I'm running 0.11.475.0, katcr.co is up and working, and I'm getting the same error about ascii characters.

**Comment by IlmLV** [comment]
Still same issue for me on Jackett Version 0.11.475.0

**Comment by garfield69** [comment]
@IlmLV  @tbotcotw  please use your jackett dashboard and edit the kickasstorrent config.
are you seeing the cookie box or usename/password box in the config form?
Also, what flavour of jackett are you running (AMDx64, ARM32, ARM64, MacOS, Mono, Windows, docker, Syno)?

**Comment by tbotcotw** [comment]
There is no cookie or password box.

I'm running AMDx64.


![Screen Shot 2019-07-09 at 2 50 06 PM](https://user-images.githubusercontent.com/42870759/60922064-24851000-a259-11e9-86c1-4277b7215fb1.png)

**Comment by cybercar** [comment]
Same issue with Jackett Version 0.11.463.0

**Comment by garfield69** [comment]
@cybercar  update to jackett 0.11.475

**Comment by garfield69** [comment]
@tbotcotw  from your jackett dashboard, delete the kickasstorrent indexer using the trashbin icon, and add anew and see if that sorts it out.

**Comment by tbotcotw** [comment]
If I delete it and recreate it the test works once, then fails every time.

**Comment by garfield69** [comment]
@tbotcotw  in that case I do not know what is going on. works for me on my win10.
assigning ticket to the c# devs @kaso17  @flightlevel  any clues what is happening here?

**Comment by cybercar** [comment]
> @cybercar update to jackett 0.11.475

Updated to Jackett Version 0.11.476.0 with same issue

**Comment by cybercar** [comment]
![jackett error](https://user-images.githubusercontent.com/23700579/60923965-579ec200-a2a0-11e9-8218-92a7f46790dc.png)

**Comment by Arzte** [comment]
I'm still running into the same issue on 0.11.476.0 on Linux. You can remove it from the list, but after you add it back it'll only work for one test before failing with the same Request headers error.

**Comment by IlmLV** [comment]
@garfield69, after update to v0.11.475 cookie field is not showing anymore and also there is no user/pass fields, 
previously tried to use v0.11.471 (or around that version), and there were cookie field, but issue was the same - with typed in cookie value or without

also tried removing indexer and adding it back, but it does not solve this problem, and also noticed that, before update, after removing indexer and adding it back cookie field was kept filled in - felt like there wasn't actual indexer removal, just hide and show

currently running on ARM32 Raspbian

**Comment by garfield69** [comment]
yeah. i can recreate the issue on my win10 VirtualBox mojave & ubuntu images.
But it works perfectly on my win10 native. 
But it is beyond my skills to debug. I've escalated it to the c# devs.

**Comment by joridiculous** [comment]
Just updated (~5 minutes ago) to 0.11.476.0. Same issue.
(Synology NAS, Intel Atom(i think) under Docker.)

**Comment by goodpunk6** [comment]
same issue. Just opened a clone of this problem on another thread and it was closed because this thread is the main tracker.

**Comment by bubez81** [comment]
same error here version Latest: v0.11.493 no cookies box when i add kickasstorrent

**Comment by jhelmink** [comment]
Same same.
Jackett Version 0.11.507.0
Configuration does not require any additional fields.
Tests after first adding the indexer fail.

**Comment by majorsl** [comment]
Ditto with 507.

**Comment by majorsl** [comment]
I fixed mine. Both the original and the .bak has high ascii characters (hence the error message.) I deleted from the gui, but noticed that the kickasstorrent.json.bak file was left behind.  After trashing the .bak and re-adding the indexer via the GUI, I no longer got the error message.

**Comment by garfield69** [comment]
when/if the c# devs resolve this issue they will post here a notice that jackett version 0.X.xxx contains the fix.
until then, other than registering your interest in this ticket, there is little point in posting updates every time another jackett release is published, stating that the problem is still outstanding, because duh! we know. ;-)

**Comment by grillp** [comment]
I also have the issue. v0.11.534.0 -> Synology DSM / Docker image -> https://hub.docker.com/r/linuxserver/jackett/

**Comment by Sigri44** [comment]
Why new issues is duplicate, and here this is closed but not solved.... ?!

**Comment by gacpac** [comment]
This is annoying I can't use kickass

**Comment by tbotcotw** [comment]
The ticket was reopened when they realized they didn't fix Linux.

**Comment by gacpac** [comment]
for now I'm using katli I guess is a workaround for all I need.

**Comment by MammothJerk** [comment]
Same issue

**Comment by MoathS** [comment]
I faced the same issue on my Unraid server using 6.7.2 binhex jackett, simply deleted and add the indexer again and it's working now.

**Comment by Noctam** [comment]
Hello,

Same issue here with Current Jackett: v0.11.655 Latest: v0.11.655 on Vm Esxi

**Comment by arlllk** [comment]
Maybe is the new update to mono 6, I was having the same problem, but when I downgraded to mono 5, it stopped

**Comment by Sigri44** [comment]
Hello,

Same issue here with Current Jackett: v0.11.723 on Docker

**Comment by lukjod** [comment]
Same issue here with Current Jackett: v0.11.723 on Docker on qnap aarch64 linux

**Comment by STaRDoGG** [comment]
How do I get that Cookie field? I'm using Jackett Version 0.12.1084.0, and it doesn't even show it?

![image](https://user-images.githubusercontent.com/1524526/70210417-31b70580-16f8-11ea-8823-3e0dc90d97f0.png)

**Comment by garfield69** [comment]
@STaRDoGG  this site is public and does not require a login, so you will not see a cookie box when you edit the config from your jackett dashboard.

the cookie login method was enabled briefly for this site;s jackett indexer as an attempt (when the problem first appeared) to try to bypass the issue. But it did not work so it was revoked.

Subsequent analysts revealed that when you send a http search request to the site, it send back the results along with some cookies values which regular Browsers like Chrome or Firefox can handle, and which Jackett on Windows can handle, but which Jackett on the Linux platforms does not.
Since the jackett source code is the same for all platforms, it is likely that the problem has something to so with the Linux specific .core net libraries that Jackett calls on.

and that is about as far as the research has got on this ticket. Nothing that you can do.

**Comment by STaRDoGG** [comment]
Aha! Thanks for taking time to explain it @garfield69 , appreciate it! =)

**Comment by IlmLV** [comment]
Hey everyone!
looks like problem was solved on tracker side, suddenly indexer started to work without making any changes or Jackett update

**Comment by garfield69** [comment]
excellent.

**Comment by joridiculous** [comment]
Exact same problem still here. Latest jacket.

**Comment by garfield69** [comment]
yeah. I still get this on my test images of  mojave macos and my ubuntu18

**Comment by ngosang** [comment]
@garfield69 I'm on Linux and this site always has been working for me Do you have detailed steps/searches to replicate the issue?

**Comment by garfield69** [comment]
I just click the test icon for the indexer and get the error message as described in the title.
But I have stumbled on a possible solution.
I just now tried it on my ubuntu18 image, and got the expected error when using the test button.
so then I clicked on the trashbin icon to remove the indexer, and accessed the add-indexer panel and added the indexer back to my active page.
and for now, the indexer is working again ;-)
I have repeated this sequence on my mojave image and that is now working to.
I think @IlmLV was right, kat have changed the cookies they use on their server and users of jackett with this problem just have to remove the indexer and add it again and they should be good to go.
I am closing this ticket.
